src/capi-single-supported/index.js

```diff
@@ -0,0 +1,73 @@
+import config from '../_shared/js/config';
+import { enableToggles } from '../_shared/js/ui.js';
```

Comment:
I'm now hard-coding in the endpoint following our email chat, so is config still necessary?